.github/workflows/update-deb-package-snapshots.yml

```diff
@@ -29,13 +29,43 @@ jobs:
           bazel --bazelrc=$GITHUB_WORKSPACE/.github/workflows/ci.bazelrc fetch //private/pkg/test/oci_image:test_sboms && break || sleep 20;
         done
         bazel run @//private/pkg/test/oci_image:test_sboms
-    
+
+    - name: Create commits
+      run: |
+        # Check work tree and end successfully if empty
+        if [ -z "$(git status --porcelain)" ]; then
+          echo "Nothing to commit";
+          exit 0;
```

Comment:
feel like this should set an output, and the step `Create Pull Request` should run conditionally on that using `if`, that would negate the need to check lines 57-60

Reply:
yeah it makes sense. Will do that!

Reply:
Done. I've also tested if the if clause is working properly with https://github.com/joycebrum/distroless/actions/runs/8237496450/job/22526374534 and https://github.com/joycebrum/distroless/actions/runs/8237449333

Reply:
@loosebazooka PTL